packages/ui/src/views/pull-request/pull-request-list-header-title.tsx

```diff
@@ -0,0 +1,60 @@
+import { cn } from '@utils/cn'
+
+import { Icon } from '../../components/icon'
+import { Text } from '../../components/text'
+
+export const PullRequestListHeaderTitle = ({
+  setHeaderFilter,
+  headerFilter,
+  closed_prs,
+  open_prs
+}: {
+  setHeaderFilter: (state: string) => void
+  headerFilter: string
+  closed_prs?: number
+  open_prs?: number
```

Comment:
Can we keep to camelcase for props here too for consistency?
```suggestion
  closedPRs?: number
  openPRs?: number
```

Reply:
done